libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -55,6 +57,20 @@ import {
   getSum,
 } from './utils/radial-chart-utils';
 
+/**
+ * @internal
+ * Output type for hoverStart and hoverEnd attributes.
+ * Contains the main data of the hovered series, and wether it was hovered in the pie or on the legend.
+ */
```

Comment:
```suggestion
/**
 * Output type for hoverStart and hoverEnd attributes.
 * Contains the main data of the hovered series, and wether it was hovered in the pie or on the legend.
 */
```

Reply:
Thank you! Just updated my PR with your comments :) Thank you for your help!